src/views/index.html

```diff
@@ -29,8 +29,15 @@ <h1>
     {% if dataset_select is not defined %}
     <p>Wildcards can be used in specified fields. Use _ for single characters or % for multiple characters.</p>
     <p><i>E.g. ("Miss_ng" will find "Missing", "Miss%" will find "Missing")</i></p>
-    {% else %}
-    <p><i><u>Note:</u> Strict searching must be checked for searches using non-fuzzy fields (e.g. Badge)</i></p>
+    {% elif allow_strict_search %}
+    <p>
+        <i>
+            <u>Note:</u>
+            Strict searching must be checked for searches using non-fuzzy fields
+            {# put this all on one line to avoid adding extra spaces :( #}
+            ({% for entity in strict_entities %}{{ entity.entity_name }}{% if not loop.last %}, {% else %}){% endif %}{% endfor %}
```

Comment:
I'm not sure how familiar you are with Jinja, but it does provide [whitespace controls](https://jinja.palletsprojects.com/en/3.0.x/templates/#whitespace-control) for situations like this! I never know where to put them :joy: but I messed around a bit and this looks like it produces the same results!

```suggestion
            (
            {%- for entity in strict_entities -%}
                {{ entity.entity_name }}
            {%- if not loop.last %}, {%- else %}){% endif %}
            {% endfor -%}
```

Reply:
> I'm not sure how familiar you are with Jinja

Not that familiar! 😅 

And awesome! Thanks for teaching me about this. I'll update this with your code.